portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -116,13 +95,11 @@ private int getUserChoice(HttpServletRequest request) {
     try {
       userChoice = Integer.parseInt(userChoiceString);
     } catch (NumberFormatException e) {
-      System.err.println("Could not convert to int: " + userChoiceString);
       return 5;
     }
 
-    // Check that the input is between 1 and 3.
+    // Check that the input is between 1 and 20.
     if (userChoice < 1 || userChoice > 20) {
```

Comment:
If we expect to update the number of choices in the future then it may be worthwhile to change this to a constant value so it can quickly referenced/updated later. 

For example:
```
public static final int MAX_USER_CHOICES = 20;
...
if (userChoice <= 0 || userChoice > MAX_USER_CHOICES) {
```